code/API_definitions/IoT-device-communication-function-management.yaml

```diff
@@ -0,0 +1,732 @@
+openapi: 3.0.3
+info:
+  title: CAMARA IoT Device Communication Function Management

```

Comment:
Please use here only the defined API name as defined in line 121. If it is `device-communication-function` then use here "Device Communication Function".

Even more important is to name the file `device-communication-function.yaml` it has to aligned with the API name in line 121.